x-pack/qa/kerberos-tests/src/test/java/org/elasticsearch/xpack/security/authc/kerberos/SpnegoHttpClientConfigCallbackHandler.java

```diff
@@ -206,8 +206,8 @@ public synchronized LoginContext login() throws PrivilegedActionException {
         try {
             return AccessController.doPrivileged((PrivilegedExceptionAction<T>) () -> Subject.doAsPrivileged(subject, action, acc));
         } catch (PrivilegedActionException pae) {
-            if (pae.getCause() instanceof PrivilegedActionException) {
-                throw (PrivilegedActionException) pae.getCause();
+            if (pae.getCause()instanceof PrivilegedActionException privilegedActionException) {
```

Comment:
reinstate space before instanceof

Reply:
@ywelsch unfortunately, that whitespace missing there is the result of spotless, because of a bug: https://bugs.eclipse.org/bugs/show_bug.cgi?id=574437. We are waiting for Spotless to fix it on their side: https://github.com/diffplug/spotless/issues/1046.

Reply:
ok 😞